app/assets/config/miq_plugin_example_manifest.js

```diff
@@ -0,0 +1,2 @@
+//= link_directory ../javascripts/red_hat_cloud_services .js
```

Comment:
Is this file needed? The file name is plugin_example.

Reply:
@PanSpagetka : can you remove this?

Reply:
@PanSpagetka Please also do this in your followup